src/components/Tooltip/TooltipRenderedOnPageBody.js

```diff
@@ -148,49 +140,34 @@ class TooltipRenderedOnPageBody extends React.PureComponent {
         } else {
             content = (
                 <Text numberOfLines={this.props.numberOfLines} style={tooltipTextStyle}>
-                    <Text
-                        style={tooltipTextStyle}
-                        ref={(ref) => {
-                            // Once the text for the tooltip first renders, update the width of the tooltip dynamically to fit the width of the text.
-                            // Note that we can't have this code in componentDidMount because the ref for the text won't be set until after the first render
-                            if (this.textRef) {
-                                return;
-                            }
-
-                            this.textRef = ref;
-                            this.updateTooltipContentWidth();
-                        }}
-                    >
+                    <Text style={tooltipTextStyle}>
                         {this.props.text}
                     </Text>
                 </Text>
             );
         }
 
-        const isCustomContent = _.isFunction(this.props.renderTooltipContent);
-
         return ReactDOM.createPortal(
-            <>
-                {/* If rendering custom content always render an invisible version of
-                    it to detect layout size changes, if the content updates. */}
-                {isCustomContent && (
-                    <View
-                        style={styles.invisible}
-                        onLayout={this.updateTooltipContentWidth}
-                    >
-                        {this.props.renderTooltipContent()}
-                    </View>
-                )}
-                <Animated.View
-                    onLayout={this.measureTooltip}
-                    style={[tooltipWrapperStyle, animationStyle]}
+            <Animated.View
+                onLayout={this.measureTooltip}
+                style={[tooltipWrapperStyle, animationStyle]}
+            >
+                <View
```

Comment:
Wrapping the content is another View is not necessary nor wanted.

Reply:
It is needed. This view will measure the width of the content, regardless of whether the content is text or custom content.

Reply:
Yeah I got that, what I meant is that maybe we don't need to wrap inside a new View. But given it may actually reduce the logic it's probably a good tradeoff. Just need to test against tooltips with prop `absolute={true}`